json2html/jsonconv.py

```diff
@@ -35,30 +35,21 @@
 
 class Json2Html:
 
-    def convert(self, **kwargs):
+    def convert(self, json="", table_attributes='border="1"'):
         """
             Convert JSON to HTML Table format
         """
 
         # table attributes such as class, id, data-attr-*, etc.
         # eg: table_attributes = 'class = "table table-bordered sortable"'
-        table_attributes = None
-        if 'table_attributes' in kwargs:
-            table_attributes = kwargs['table_attributes']
-        else:
-            # by default HTML table border
-            table_attributes = 'border="1"'
         self.table_init_markup = "<table %s>" % table_attributes
 
-        json_input = None
-        if 'json' in kwargs and kwargs['json']:
-            unvalidated_input = kwargs['json']
-            if type(unvalidated_input) in text_types:
-                json_input = json.loads(unvalidated_input, object_pairs_hook=OrderedDict)
-            else:
-                json_input = unvalidated_input
+        if not json:
+            json_input = {}
+        elif type(json) in text_types:
+            json_input = json_parser.loads(json, object_pairs_hook=OrderedDict)
```

Comment:
I think we should discuss what should be the correct behaviour here. 

Do we want the user to know that he didn't submit correct JSON? Or should we just do a best effort to produce something sensible?

What if user does something like

`json2html.convert('string: [1, 2, 3]')`

? In that case we can at best just return his string, which isn't really what is expected of this library.

So after doing some thinking, IMO a ValueError is actually the correct behaviour, because then at least it's clear what went wrong.